docs/secondary-navigation.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: sub-navigation
+order: 5
+title: Secondary navigation
+description: Allow users to navigate between pages scoped to a section.
+---
+
+{% from "example/macro.njk" import appExample %}
+
+{{ appExample({
+  component: "secondary-navigation"
+}) }}
+
+## When to use this component
+
+Use the secondary navigation when you already have a [primary navigation](primary-navigation) and need to let users navigate between pages that are scoped to a particular section or concept.
+
+For example, if your service is a case management system, you might use the secondary navigation if there are multiple pages associated with a single case.
+
+## When not to use this component
+
+Do not use this component for the primary sections of your service, or if a [sub-navigation](sub-navigation) would be more appropriate.
+
+You could also consider using the [tabs component](https://design-system.service.gov.uk/components/tabs/) if the amount of content in each sub-section is small.
+
+## How it works
+
+{{ appExample({
+  component: "secondary-navigation",
+  example: "with-header"
+}) }}
+
+Unlike [tabs](https://design-system.service.gov.uk/components/tabs/), secondary navigation does not use JavaScript. Each item within the navigation is a link to a separate full page.
+
+One of the items should always be highlighted as the current page. This item remains a link, but uses black text and has a line beneath it.
+
+The secondary navigation should appear before the `<main>` element of the page. This means that it is skipped when [skip link](https://design-system.service.gov.uk/components/skip-link/) is used.
+
+The title of the current page can be repeated below the sub navigation, inside the `<main>` element, using a heading tag.
+
+You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+### Nunjucks macro options
```

Comment:
Tempted to update documentation for the other components to use this heading. Can do so in a separate PR. 

```suggestion
### Nunjucks macro options
```

Reply:
Yeah, I thought it was worth following the term used on the GOV.UK Design System website (although there it's hidden away behind a Details, which I often forget about).